CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 [authState]: https://github.com/okta/okta-auth-js#authstatemanager
 
+# 3.1.0
+
+### Features
+
+- [#39](https://github.com/okta/okta-vue/pull/39) Adds support for Typescript
```

Comment:
Is this not a breaking change?  I understand that the API isn't changed, in theory, but anyone using TS with our code now has to make changes, right?

Reply:
Good question, before they are using third party types which is not maintained by us. So from the official usage standpoint, it's a new feature. 

Also, the third-party types should only work with okta-vue@2.x, and introduce our own types starts from 3.x. 

I feel it's ok to list the change as a minor update.

Reply:
But if they were using a third party type, this will break them, yes?   Officially we might not need a major version bump, but if it is less disruptive to our customers, why not acknowledge it?

Reply:
third-party types (https://www.npmjs.com/package/@types/okta__okta-vue) has already been broken since 3.0. This change is 3.1.

I agree we should mention self added types might be broken in readme, I just don't really feel a major version is needed.

Reply:
I'm not as confident.  We broke their typing once in our major release (which was a major release, cool), but if we're going to break anyone that adapted to that in a non-major release, that's still a break.  Let's chat further outside of this review and see if you can convince me - I don't know how fragile third (or second!) party typing would be.